janitor/functions/pivot.py

```diff
@@ -110,6 +109,31 @@ def pivot_longer(
         0   1        sp      m  5564      2
         1   1       rel      f    65      3
 
+    Use multiple `.value` to reshape dataframe:
+
+        >>> df = pd.DataFrame(
+        ...     [
+        ...         {
+        ...             "x_1_mean": 1,
+        ...             "x_2_mean": 1,
+        ...             "y_1_mean": 1,
+        ...             "y_2_mean": 1,
+        ...             "unit": 1,
+        ...         }
+        ...     ]
+        ... )
+        >>> df
+           x_1_mean  x_2_mean  y_1_mean  y_2_mean  unit
+        0         1         1         1         1     1
+        >>> df.pivot_longer(
+        ...     index="unit",
+        ...     names_to=(".value", "time", ".value"),
+        ...     names_pattern=r"(x|y)_([0-9])(_mean)",
+        ... )
+           unit time  x_mean  y_mean
+        0     1    1       1       1
+        1     1    2       1       1
+
 
 
     :param df: A pandas DataFrame.
```

Comment:
Can you rearrange the parameters of `pivot_longer` in the docstring below to match the order in the function signature?
In the function signature, the order is `(..., names_to, values_to, ...)` but in the docstring arglist it is not.